Docs/topics/discord/how-to-connect.md

```diff
@@ -4,7 +4,7 @@ channelId: 1144875410786623568
 
 # Server Information
 
-LittleBigRefresh currently supports LBP1, LBP2, and LBP Vita. LBP3 is currently experimental, requiring extra steps to work.
+LittleBigRefresh currently supports LBP1, LBP2, LBP Vita, and LBP PSP. LBP3 is currently experimental, requiring extra steps to work.
```

Comment:
I think LBP3 should be fine now, no?

Reply:
In general the base gameplay works now yeah, i'll reword it to remove the "experimental" tag